pkg/models/alert.go

```diff
@@ -111,6 +111,18 @@ func (this *Alert) ContainsUpdates(other *Alert) bool {
 	return result
 }
 
+func (alert *Alert) GetTagsFromSettings() []*Tag {
```

Comment:
Would like to see some tests here to verify the json schema of the alertRuleTags.

Reply:
Done